webapp/src/main/webapp/test/output.test.js

```diff
@@ -18,9 +19,116 @@ describe('Output', function() {
   });
 
   it('should issue column warning', function() {
-      unknownColumnWarning();
-      // TODO: Update the test to be dependent on the notice once unknownColumnWarning uses input notice.
-    expect(document.getElementById('warning').innerHTML).toContain('<p class="warning">Warning - Unknown Column(s) found!</p><p><b>1</b> unknown column(s) found in:</p><p>Filename: <i>hello.txt</i></p><p>Column name: <i>column name</i> at position <i>5</i></p><p>Please delete or rename column!</p>');
+    const params = {
+      code: "unknown_column",
+      totalNotices: 1,
+      notices: [
+        {
+          filename: "stop_times.txt",
+          fieldName: "drop_off_time",
+          index: 8
+        }
+      ]
+    };
+    unknown_column(params);    // Unknown column output
+    const output = "<div><p class=\"warning\">Warning - Unknown Column(s) found!</p>\
+<p>Description: A column name is unknown.</p>\
+<p><b>1</b> unknown column(s) found in:</p>\
+<table><thead><tr><th>Filename</th><th>Field name</th><th>Index</th></tr></thead><tbody>\
+<tr><td>stop_times.txt</td><td>drop_off_time</td><td>8</td></tr>\
+</tbody></table><p>Please delete or rename column!</p><br><br></div>"
+    expect(document.getElementById('warning').innerHTML).toContain(output);
+  });
+  
+  it('should issue invalid row length warning', function() {
+    const params = {
+      code: "invalid_row_length",
+      totalNotices: 2,
+      notices: [
+        {
+          filename: "stop_times.txt",
+          csvRowNumber: 17,
+          rowLength: 5,
+          headerCount: 9
+        },
+        {
+          filename: "stop_times.txt",
+          csvRowNumber: 18,
+          rowLength: 5,
+          headerCount: 9
+        }
+      ]
+    };
+    invalid_row_length(params);
+    const output = "<div><p class=\"error\">Error - Invalid csv row length!</p>\
+<p>Description: A row in the input file has a different number of values than specified by the CSV header.</p>\
+<p><b>2</b> Invalid row length found in:</p><table><thead><tr><th>Filename</th><th>CSV Row Number</th>\
+<th>Row length</th><th>Header count</th></tr></thead><tbody><tr><td>stop_times.txt</td>\
+<td>17</td><td>5</td><td>9</td></tr><tr><td>stop_times.txt</td><td>18</td><td>5</td>\
+<td>9</td></tr></tbody></table><p>Please set the row length as specified by the CSV header!</p><br><br></div>"
```

Comment:
Can you reformat this lines to make it more readable (Single line per row)?

Reply:
Yep ok

Reply:
Done. I've reformatted the lines so be a lot more readable.